src/drive/mobile/containers/settings/Support.jsx

```diff
@@ -40,11 +40,7 @@ export const Support = ({
         onClick: async () => {
           if (isOnline()) {
             try {
-              await logInfo(
-                t('mobile.settings.support.logs.title'),
-                serverUrl,
-                true
-              )
+              await logInfo(t('mobile.settings.support.logs.title'), serverUrl)
```

Comment:
do we still have to `await` the loginfo instruction's execution?

Reply:
As logInfo returns a promise, I guess it's better to await it, but honestly I'm not sure it was necessary to return a promise from the `logInfo` function...